druid/src/widget/image.rs

```diff
@@ -27,6 +23,17 @@ use crate::{
 };
 
 /// A widget that renders an Image
+///
+/// `druid` does not support loading image data by default.
+/// However, there is an optional dependency on the `image` crate
+/// that makes loading simple. This feature can be enabled with cargo.
+/// For example, in `Cargo.toml`:
+/// ```no_compile
+/// [dependencies.druid]
+/// version = "0.6.0"
+/// features = ["image"]
+/// ```
```

Comment:
I think the documentation for `Image` / `ImageData` should mention the `image` feature just briefly and link to a more detailed description how to use features elsewhere.

I think adding a section to the root of the crate (`lib.rs`) is a fine place. Just before the `Examples` section for example. The new section could list all the features we have and a brief description of what they do. Then below this list we could have a simple `Cargo.toml` example how to enable a feature.